server/app-config.ts

```diff
@@ -23,7 +23,8 @@ const isOfficeQuotas = (input: any): input is OfficeQuota[] =>
       typeof o === 'object' &&
       o !== null &&
       typeof o.name === 'string' &&
-      typeof o.quota === 'number'
+      typeof o.quota === 'number' &&
+      typeof o.parkingQuota === 'number'
```

Comment:
We should default this to zero if not provided in the ENV (after merging/rebasing with changes on master)